stub/src/main/java/io/grpc/stub/CallStreamObserver.java

```diff
@@ -95,4 +95,18 @@
    * @param enable whether to enable compression.
    */
   public abstract void setMessageCompression(boolean enable);
+
+  static final Object KEY_FOR_20171102 =
+      "I know what I'm doing and make my own observers. Today's password is actinide.";
+
+
+  /**
+   * Do not call this, it is for the gRPC library to construct.
+   */
+  @Deprecated
+  public CallStreamObserver(Object key) {
+    if (!KEY_FOR_20171102.equals(key)) {
```

Comment:
If use `==` here, then users are not able to create it outside io.grpc.stub package.